app/src/main/java/com/canvearth/canvearth/MapsActivity.java

```diff
@@ -212,7 +218,14 @@ public void onClickAddSketch() {
     }
 
     public void onClickShowSketch() {
-        ScreenUtils.showToast(this, "Not Implemented Yet");
+        SketchShowFragment fragment = (SketchShowFragment) getFragmentManager().findFragmentById(R.id.sketch_view);
+        processNearbySketches(fragment);
+        findViewById(R.id.sketch_view).setVisibility(View.VISIBLE);
+        findViewById(R.id.all_components).setVisibility(View.GONE);
+    }
+
+    public void showAllComponents() {
+        findViewById(R.id.all_components).setVisibility(View.VISIBLE);
```

Comment:
Is this needed for showing buttons again after return from `showSketch`?
If so, I think it is unnecessary for now because I implemented {in}visible all_components by tapping the screen.

Reply:
I think that automatically showing is more natural than intentionally tapping.
However, I will follow you and @YoungsooJang 's choice about all ui-related issues.
After this PR, please fix this as you think.